examples/scripts/bco.py

```diff
@@ -18,6 +18,8 @@
 # Full training:
 python examples/scripts/bco.py \
     --model_name_or_path=nnheui/stablelm-2-1_6b-sft-full \
+    --trust_remote_code \
+    --dataset_name trl-lib/ultrafeedback-gpt-3.5-turbo-helpfulness \
```

Comment:
I wonder if we can switch this to an unpaired version of `trl-lib/ultrafeedback_binarized`? That way we have two simple datasets that should "just" work

Reply:
Good idea. How `trl-lib/ultrafeedback_binarized` is generated? Which "aspect" (helpfulness, ...) is used?

Reply:
I merge the PR, but we can work on it in a following PR